### PR TITLE
fixing bugs

### PR DESCRIPTION
**Pull request by joshuaobinna**

### CODE REVIEW

File: src/components/aboutUs/constants.js [hunk 1]
```diff
@@ -68,13 +68,13 @@ export const TEAM_INFO = [
...
-            { icon: 'linkedin', link: '/https://www.linkedin.com/in/okemmadu-eric-1a3818113/' },
+            { icon: 'facebook', link: 'http://www.facebook.com/eric.okemmadu/' },
+            { icon: 'github', link: 'http://www.github.com/EngrEric/' },
+            { icon: 'linkedin', link: 'https://www.linkedin.com/in/okemmadu-eric-1a3818113/' },
         ],
     },
 ];
 
-export const MARKET_SCENES = [ '/sales.jpg', '/omil.jpg', '/farm.jpg', '/mark.jpg', '/fish.jpg' ];
+export const MARKET_SCENES = [ 'a.jpg','b.jpg','c.jpg','d.jpg','e.jpg','f.jpg','g.jpg','j.jpg' ];
```
Comment (enyeInc) [inline src/components/aboutUs/constants.js]: Can you rename these image file name?

File: src/components/aboutUs/constants.js [hunk 2]
```diff
@@ -68,13 +68,13 @@ export const TEAM_INFO = [
         name: "Okemmadu Eric",
         photo: '/okemmadu.jpg',
         social_media: [
-            { icon: 'facebook', link: '/www.facebook.com/eric.okemmadu/' },
-            { icon: 'github', link: '/github.com/EngrEric/' },
-            { icon: 'linkedin', link: '/https://www.linkedin.com/in/okemmadu-eric-1a3818113/' },
+            { icon: 'facebook', link: 'http://www.facebook.com/eric.okemmadu/' },
+            { icon: 'github', link: 'http://www.github.com/EngrEric/' },
```
Comment (enyeInc) [inline src/components/aboutUs/constants.js]: change all the `http` to `https`

**Comment by EngrEric** [comment]
Looks good to me but it will be more better if the links open on a new tab allowing users not to exit our website.

**Comment by Simpcyclassy** [comment]
Looks good to me

**Pull request merged**